google_compute_engine/instance_setup/tests/instance_setup_test.py

```diff
@@ -325,21 +328,23 @@ def testSetSshHostKeysFirstBoot(self, mock_listdir):
     self.mock_setup._GenerateSshKey = mock_generate_key
     mock_listdir.return_value = [
         'ssh_config',
-        'ssh_host_rsa_key',
+        'ssh_host_dsa_key',
```

Comment:
Should be ssh_host_ecdsa_key if we are doing that by default.
Older distros still have dsa keys (EL6, Deb8)

Reply:
These aren't default, this is a made up list of strings for testing. The types may as well be "apple" "puppy" "cookie". This is just verifying that the existing files on the system (with names different from the names in the config list) will also be regenerated.